taxonomy/management/commands/refresh_job_postings_data.py

```diff
@@ -17,10 +17,12 @@
 
 class Command(BaseCommand):
     """
-            Example usage:
-            Create or refresh the existing data associated with jobs and job postings on the basis
-            of ranking_facet.
-            $ ./manage.py refresh_job_postings_data 'TITLE_NAME'
+    Command to refresh the job postings data associated with a job.
+
+    Example usage:
+        $ # Create or refresh the existing data associated with jobs and job postings on the basis
+        $ # of ranking_facet.
```

Comment:
I think we need to remove `#` character from both the above lines?

Reply:
I added them to indicate that these are comments and not the command, bash comments start with `#`